postcode_lookup/template_sorter.py

```diff
@@ -162,7 +165,34 @@ def toc_label(self):
 
     @property
     def toc_id(self):
-        return "voter-registration"
+        return f"voter-registration-{self.timetable.registration_deadline}"
+
+
+class CityOfLondonRegistrationDateSection(RegistrationDateSection):
+    template_name = "includes/registration_timetable_col.html"
+
+    def __init__(self, *args, **kwargs) -> None:
+        self.with_headers = kwargs.pop("with_headers")
```

Comment:
Basically: If we're showing 2 "register to vote" CTAs, we need a way to say "don't show the headers on the second one"